fn main() {
    let number;

    number = 12;

    println!("{}", number); 

    // this will generate an error
    let number1;

    number1 = 384;
    number1 = 382;

    println!("{}", number1);
}

